scalapack-targets.cmake path to libscalapack.a or libscalapack.dll

### Description / Steps to reproduce the issue

I try to build MFEM packages with MUMPS but I have this error:

```
-- MUMPS: looking for required package: ScaLAPACK
CMake Error at C:/msys64/mingw64/lib/cmake/scalapack-2.1.0/scalapack-targets.cmake:84 (message):
  The imported target "scalapack" references the file
     "C:/msys64/mingw64/Testing/libscalapack.dll"
  but this file does not exist.  Possible reasons include:
  * The file was deleted, renamed, or moved to another location.
  * An install or uninstall procedure did not complete successfully.
  * The installation package was faulty and contained
     "C:/msys64/mingw64/lib/cmake/scalapack-2.1.0/scalapack-targets.cmake"
  but not all the files it references.
```

This is strange: `C:/msys64/mingw64/Testing/libscalapack.dll`


### Expected behavior

`C:/msys64/mingw64/bin/libscalapack.dll`

![image](https://github.com/msys2/MINGW-packages/assets/24394573/209261c4-1605-4681-8978-c26b98d368a4)


### Actual behavior

`C:/msys64/mingw64/Testing/libscalapack.dll`

### Verification

- [X] I have verified that my MSYS2 is up-to-date before submitting the report (see https://www.msys2.org/docs/updating/)

### Windows Version

MINGW64_NT-10.0-22621

### MINGW environments affected

- [X] MINGW64
- [ ] MINGW32
- [ ] UCRT64
- [ ] CLANG64
- [ ] CLANG32
- [ ] CLANGARM64

### Are you willing to submit a PR?

_No response_

--- a/mingw-w64-scalapack/0002-scalapack-fix-bindir.patch
+++ b/mingw-w64-scalapack/0002-scalapack-fix-bindir.patch
@@ -1,0 +1,20 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -85,7 +85,7 @@
+   install(TARGETS ${lib} EXPORT scalapack-targets
+     ARCHIVE DESTINATION lib${LIB_SUFFIX}
+     LIBRARY DESTINATION lib${LIB_SUFFIX}
+-    RUNTIME DESTINATION Testing
++    RUNTIME DESTINATION bin
+   )
+ endmacro()
+ 
+@@ -100,7 +100,7 @@
+ 
+ # Organize output files.  On Windows this also keeps .dll files next
+ # to the .exe files that need them, making tests easy to run.
+-set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${SCALAPACK_BINARY_DIR}/TESTING)
++set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${SCALAPACK_BINARY_DIR}/bin)
+ set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${SCALAPACK_BINARY_DIR}/lib)
+ set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${SCALAPACK_BINARY_DIR}/lib)
+ 

--- a/mingw-w64-scalapack/PKGBUILD
+++ b/mingw-w64-scalapack/PKGBUILD
@@ -4,7 +4,7 @@ _realname=scalapack
 pkgbase=mingw-w64-${_realname}
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
 pkgver=2.2.0
-pkgrel=2
+pkgrel=3
 arch=('any')
 mingw_arch=('mingw64' 'ucrt64' 'clang64')
 pkgdesc="Scalable Linear Algebra PACKage (mingw-w64)"
@@ -20,10 +20,12 @@ license=('BSD')
 url="https://www.netlib.org/scalapack"
 source=("https://www.netlib.org/${_realname}/${_realname}-${pkgver}.tgz"
         '0001-fix-building-with-flang.patch'
+        '0002-scalapack-fix-bindir.patch'
         '0003-force-single-lib.patch'
         '0004-fix-pc.patch')
 sha256sums=('40b9406c20735a9a3009d863318cb8d3e496fb073d201c5463df810e01ab2a57'
             'cecb30a91e059c1b61628ed45c8cb5a0fdc865b5594016d443719b3bba1b5380'
+            '9d763267c1dbe494eef426d7d662cf5da736b96693ee51c9d2c099167e179662'
             '954c22f1107a254a46400a0014f17af9c5b8877f0034793e2c6781eafafa8225'
             '9f791fdb6a6d59cd86caa53cec62a1a6dbb2f6f31bd16f8320864615695e175e')
 
@@ -42,6 +44,7 @@ prepare() {
       '0001-fix-building-with-flang.patch'
   fi
   apply_patch_with_msg \
+    '0002-scalapack-fix-bindir.patch' \
     '0003-force-single-lib.patch' \
     '0004-fix-pc.patch'
 }
@@ -86,10 +89,8 @@ build() {
 }
 
 package() {
-  # FIXME proper cmake configuration
   cd "${srcdir}/build-${MSYSTEM}s"
   DESTDIR="${pkgdir}" ${MINGW_PREFIX}/bin/cmake --install .
   cd "${srcdir}/build-${MSYSTEM}"
   DESTDIR="${pkgdir}" ${MINGW_PREFIX}/bin/cmake --install .
-  cd "${pkgdir}/${MINGW_PREFIX}" && mv Testing bin
 }

